omou-frontend/src/reducers/initialState.js

```diff
@@ -1,3 +1,481 @@
+                ],
+                "Parent Information": [
+                    {
+                        field: "Parent Name",
+                        name: "Parent Name",
+                        type: "short text",
+                    },
+                    {
+                        field: "Relationship to Student",
+                        name: "Relationship to Student",
+                        type: "short text",
+                    },
+                    {
+                        field: "Parent Email",
+                        name: "Parent Email",
+                        type: "email",
+                    },
+                    {
+                        field: "Parent Phone Number",
+                        name: "Phone Number",
+                        type: "phone number",
+                    },
+                    {
+                        field: "Address",
+                        name: "Address",
+                        type: "address",
+                    },
+                    {
+                        field: "City",
+                        name: "City",
+                        type: "short text",
+                    },
+                    {
+                        field: "State",
+                        name: "State",
+                        type: "short text",
+                        suggestions: [{label: "AL"}, {label: "AK"}, {label: "AZ"}, {label: "AR"}, {label: "CA"}, {label: "CO"},
+                        {label: "CT"}, {label: "DE"}, {label: "FL"}, {label: "GA"}, {label: "HI"}, {label: "ID"},
+                        {label: "IL"}, {label: "IN"}, {label: "IA"}, {label: "KS"},
+                        {label: "KY"}, {label: "KY"}, {label: "LA"}, {label: "ME"}, {label: "MD"}, {label: "MA"}, {label: "MI"},
+                        {label: "MS"}, {label: "MO"}, {label: "MT"}, {label: "NE"}, {label: "NV"}, {label: "NH"}, {label: "NJ"},
+                        {label: "NM"}, {label: "NY"}, {label: "NC"}, {label: "ND"},
+                        {label: "OH"}, {label: "OK"}, {label: "OR"}, {label: "PA"}, {label: "RI"}, {label: "SC"}, {label: "SD"},
+                        {label: "TN"}, {label: "TX"}, {label: "UT"}, {label: "VT"}, {label: "VA"}, {label: "WA"}, {label: "WV"}, {label: "WI"},
+                        {label: "WY"}].map(suggestion => ({
+                            value: suggestion.label,
+                            label: suggestion.label,
+                        })),
+                    },
+                    {
+                        field: "Zip Code",
+                        name: "Zip Code",
+                        type: "string",
+                    },
+                ],
+            },
+            tutoring: {
+                form_type: "tutoring",
+                section_titles: ["Tutoring Session Type", "Student(s)", "Student Information", "Tutor Selection", "Payment"],
+                field_limits: {
+                    "Select tutoring type": 1,
```

Comment:
instead of having field_limits as its own object, just have each field object have a `field_limits` property. For example: 
```
{
    field: "City",
    name: "City",
    type: "short text",
    field_limit: 1,
},
```